### ISSUE TITLE
ADA - Color contrast does not meet minimum requirement

### ISSUE THREAD

**Issue by rbruhn** [AustinMastLab/WeDigBio-D7#13]
There is insufficient color contrast between the text and background on this page to meet the minimum contrast requirement.
218 Pages

<img width="933" height="495" alt="Image" src="https://github.com/user-attachments/assets/1619fb16-b046-48e8-b649-6272ac419310" />
